iceoryx_posh/test/integrationtests/test_service_discovery.cpp

```diff
@@ -55,316 +107,314 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
 }
 
 
-TEST_F(RoudiFindService_test, OfferServiceWithDefaultServiceDescriptionFails)
+TEST_F(ServiceDiscovery_test, OfferServiceWithDefaultServiceDescriptionFails)
 {
     ::testing::Test::RecordProperty("TEST_ID", "1db1ce50-4e95-46f3-8682-9cc90576dbc0");
-    auto isServiceOffered = senderRuntime->offerService(iox::capro::ServiceDescription());
+    auto isServiceOffered = sut.offerService(iox::capro::ServiceDescription());
     this->InterOpWait();
```

Comment:
Why wait here? I think it is not required as the result is no future or similar (and then waiting for a flat time would still be incorrect).